.github/workflows/build.yml

```diff
@@ -26,7 +26,9 @@ jobs:
           arguments: conformanceTest conformanceTestOnSamples
       - name: Check out jspecify/samples-google-prototype
         run: git -C ../jspecify checkout samples-google-prototype
+        if: always()
       - name: Run Samples Tests
         uses: gradle/gradle-build-action@v2
         with:
           arguments: jspecifySamplesTest
+        if: always()
```

Comment:
These seem like a good change to make on the main branch, separately.

Reply:
I opened #121 for this.